<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCPTZControl</class>
 <widget class="QDialog" name="VSCPTZControl">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>218</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenCVR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../velibui.qrc">
    <normaloff>:/action/resources/libuilogoc.png</normaloff>:/action/resources/libuilogoc.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="2" column="1">
            <widget class="QPushButton" name="pbDown">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptz-down.png</normaloff>:/action/resources/ptz-down.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pbUp">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptz-up.png</normaloff>:/action/resources/ptz-up.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pbLeft">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptz-left.png</normaloff>:/action/resources/ptz-left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pbRight">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptz-right.png</normaloff>:/action/resources/ptz-right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pbZoomin">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptzzoomin.png</normaloff>:/action/resources/ptzzoomin.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbZoomout">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
border:none;
color:white;
background-color: rgb(85, 170, 255);}
QPushButton:pressed {
background-color: rgb(200, 200, 200);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../velibui.qrc">
               <normaloff>:/action/resources/ptzzoomout.png</normaloff>:/action/resources/ptzzoomout.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="speedControl">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">	QSlider::groove:horizontal {
border: 1px solid #999999;
height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
}
QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
border: 1px solid #5c5c5c;
width: 18px;
margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
border-radius: 3px;
}</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../velibui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>VSCPTZControl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
